logicalplan/distribute.go

```diff
@@ -535,6 +533,9 @@ func isConstantExpr(expr parser.Expr) bool {
 	case *parser.ParenExpr:
 		return isConstantExpr(texpr.Expr)
 	case *parser.Call:
+		if len(texpr.Args) == 0 {
```

Comment:
technically not needed i think; we will already return true if tepr.Args is empty list

Reply:
Yes good catch.